### ISSUE TITLE
How to model RPS in GCAM 5.1.2?

### ISSUE THREAD

**Issue by robbieorvis** [JGCRI/gcam-core#47]
Hello,

I'm trying to model an RPS in GCAM 5.1.2. It looks like this process has changed from previous v4 versions of GCAM. Could someone provide an example or instructions?

Thanks.

**Comment by rjplevin** [comment]
As Robbie knows, I've added a feature to pygcam to allow the user to easily and flexibly create an RPS using a custom XML file. However, it requires a patch that Pralit created that users don't have access to. I can post this patch, but many users won't be able to compile GCAM themselves. I haven't undertaken the task of setting up a Windows build environment either.

It would be great to get that patch into GCAM in the next release so Robbie and others can use the new pygcam feature. Barring that, it would be great in the near term if you could share a Windows gcam.exe with the patch with Robbie.

**Comment by rjplevin** [comment]
Just to update this: I did get the Windows build environment working and posted (on gcam-doc) the updated instructions that work with the current Visual Studio (2017). So Robbie now has an executable with the patch.

I've also added the ability to translate a much simpler, CSV file describing the set of RES constraints into the necessary GCAM input file. I think this would be very useful to other users, but it requires the source code patch.
